### PR TITLE
FlyingBear profiles updates: abs filament

### PR DESCRIPTION
**Pull request by FlyingbearOfficial**

### CODE REVIEW

**Comment by FlyingbearOfficial** [comment]
Hi,I closed  previous PR and create new one for testing. Are there some problems here?

File: resources/profiles/FlyingBear/machine/fdm_klipper_common.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
   "cooling_tube_length": "5",
   "cooling_tube_retraction": "91.5",
   "default_filament_profile": [
-      "FlyingBear Generic PLA"
+      "FlyingBear PLA"
```
Comment (SoftFever) [inline resources/profiles/FlyingBear/machine/fdm_klipper_common.json:19]: You might want to revert this line? 
As discussed in previous sessions.

**Comment by SoftFever** [comment]
> Hi,I closed previous PR and create new one for testing. Are there some problems here?

Overall looks good to me other than a small issue.
Please refer to the code review comment for the detailed info

File: resources/profiles/FlyingBear/machine/fdm_klipper_common.json (hunk 1 above)
Comment (FlyingbearOfficial) [reply resources/profiles/FlyingBear/machine/fdm_klipper_common.json]: I fixed this code, please check it

**Review by SoftFever — APPROVED** [review]
Looks good.
Thanks

**Pull request merged**